oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -93,6 +95,11 @@ These steps should be followed for all repos within the edX organization(forks i
 
    - Archive the repository per `GitHub's archive process`_
 
+.. note::
+    Over the lifetime of Open edX, we may fork the same external open source repository multiple times.  In this case, we may need to archive the fork multiple times as we move between our fork and following upstream.  When this is necessary, delete the old copy of the fork before you move the new repo to edx-unsupported.
+
+    This may break some older version of Open edX.  Some combination of copying branches between forks, renaming branches, and changing unsupported versions of Open edX would have to be done to keep things working.  We opt not to take on this extra work by default thought may do so under extenuating circumstances.
+
```

Comment:
One "extenuating circumstance" that we may want to call out is: the previous fork is still being used by the previous Open edX named release. Since we are supporting 1-previous named release, we need to ensure the previous release continues to function.  In that case, one idea is to move the previous fork to the [openedx github organization](http://github.com/openedx) and rename it for easier discovery (such as `<juniper>-fork-<repo-name>`).

Reply:
I'll call that out,  just moving the old fork is not sufficient because a lot of the redirects will break but I'll add a note about it and suggest porting the branches of the old fork to the new fork.

Reply:
Late thought: What if when we made forks, we named them after the upcoming release right away? It might be easier to remember at that moment than at archiving time, avoid name conflicts, and make it clearer which ones were safe to delete.